privaterelay/views.py

```diff
@@ -133,6 +134,23 @@ def _get_fxa(request):
     return request.user.socialaccount_set.filter(provider='fxa').first()
 
 
+@api_view()
+@require_http_methods(['GET'])
+def profile_refresh(request):
```

Comment:
Hmm ... what do we need this view for? What user flow?

Reply:
The equivalent of this is currently run when server the `/accounts/profile` view, so it's the equivalent of

https://github.com/mozilla/fx-private-relay/blob/df47f8a217ddc1cf6b057bb8ac7139664976dd2f/privaterelay/views.py#L79-L81

and

https://github.com/mozilla/fx-private-relay/blob/df47f8a217ddc1cf6b057bb8ac7139664976dd2f/privaterelay/views.py#L105-L109